<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemMicroorganism</class>
 <widget class="QWidget" name="ItemMicroorganism">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemMicroorganism</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {border: 2px solid gray;}</string>
  </property>
  <widget class="QLabel" name="ItemName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {border: 0px solid gray;}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LivetimeText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {border: 0px solid gray;}</string>
   </property>
   <property name="text">
    <string>Время жизни(осталость):</string>
   </property>
  </widget>
  <widget class="QLabel" name="OffspringText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {border: 0px solid gray;}</string>
   </property>
   <property name="text">
    <string>Потомство через:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LivetimeValue">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {border: 0px solid gray;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="OffspringValue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>55</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {border: 0px solid gray;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
